avtozip/avtozip/settings.py

```diff
@@ -56,10 +56,13 @@
 
 ROOT_URLCONF = 'avtozip.urls'
 
+print(BASE_DIR)
```

Comment:
You are right, our code should not have `print` statements at all. It was debug purpose